pkg/types/openstack/platform.go

```diff
@@ -84,4 +84,8 @@ type Platform struct {
 	// The subnet and network specified in MachinesSubnet will not be deleted or modified by the installer.
 	// +optional
 	MachinesSubnet string `json:"machinesSubnet,omitempty"`
+
+	// DisableSecurityGroups will allow clusters to be created without the use of OpenStack Security Groups. Some Openstack
+	// configurations do not support security groups. This is generally not recommended where security groups are supported.
+	DisableSecurityGroups bool `json:"disableSecurityGroups,omitempty"`
```

Comment:
Is there a way for the installer to determine whether the OpenStack configuration supports security groups without requiring the user to specify this?

Reply:
I wondered about that as well, and am not sure. If someone knows, this would be a good add. Maybe just in a follow-up patch.

Reply:
Note that what I am suggesting is that if there is a way for the installer to determine this, then we should not even have it as an option for the user. In other words, we would eliminate this field.